{"name": "Проверка успешного перехода в раздел «История заказов»", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message:", "trace": "self = <test_account.TestAccount object at 0x000001A4F3F8E1E0>, driver = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"966af1de-daa3-4bb5-b893-e55e5fd7fe4e\")>\nnavigate = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"966af1de-daa3-4bb5-b893-e55e5fd7fe4e\")>\nsetup_method = (<pages.main_page.MainPage object at 0x000001A4F3FFF650>, <pages.account_page.AccountPage object at 0x000001A4F3FFFC20>, <pages.list_of_orders_page.ListOfOrdersPage object at 0x000001A4F3FFDC70>)\nlogin = <selenium.webdriver.firefox.webdriver.WebDriver (session=\"966af1de-daa3-4bb5-b893-e55e5fd7fe4e\")>\n\n    @allure.title('Проверка успешного перехода в раздел «История заказов»')\n    def test_click_history_button_shows_order_history(self, driver, navigate, setup_method, login):\n        main, account, list = setup_method\n        #создаем заказ, чтобы проверить переход в историю заказов\n        main.create_order()\n>       main.wait_invisible_number_9999_in_created_order_window()\n\ntests\\test_account.py:15: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\npages\\main_page.py:57: in wait_invisible_number_9999_in_created_order_window\n    self.invisible_element(Locators.ORDER_NUMBER_9999)\npages\\base_page.py:91: in invisible_element\n    WebDriverWait(self.driver, timeout).until(expected_conditions.invisibility_of_element_located((locator)))\n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nself = <selenium.webdriver.support.wait.WebDriverWait (session=\"966af1de-daa3-4bb5-b893-e55e5fd7fe4e\")>\nmethod = <function invisibility_of_element_located.<locals>._predicate at 0x000001A4F4071C60>, message = ''\n\n    def until(self, method: Callable[[D], Union[Literal[False], T]], message: str = \"\") -> T:\n        \"\"\"Calls the method provided with the driver as an argument until the \\\n        return value does not evaluate to ``False``.\n    \n        :param method: callable(WebDriver)\n        :param message: optional message for :exc:`TimeoutException`\n        :returns: the result of the last call to `method`\n        :raises: :exc:`selenium.common.exceptions.TimeoutException` if timeout occurs\n        \"\"\"\n        screen = None\n        stacktrace = None\n    \n        end_time = time.monotonic() + self._timeout\n        while True:\n            try:\n                value = method(self._driver)\n                if value:\n                    return value\n            except self._ignored_exceptions as exc:\n                screen = getattr(exc, \"screen\", None)\n                stacktrace = getattr(exc, \"stacktrace\", None)\n            time.sleep(self._poll)\n            if time.monotonic() > end_time:\n                break\n>       raise TimeoutException(message, screen, stacktrace)\nE       selenium.common.exceptions.TimeoutException: Message:\n\n.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py:105: TimeoutException"}, "steps": [{"name": "Оформить заказ", "status": "passed", "steps": [{"name": "Перетаскиваем выбранный ингредиент в зону формирования бургера", "status": "passed", "steps": [{"name": "Перетаскиваем выбранный элемент в нужное поле на странице", "status": "passed", "steps": [{"name": "Находим элемент", "status": "passed", "parameters": [{"name": "locator", "value": "['xpath', \".//a[(@href='/ingredient/61c0c5a71d1f82001bdaaa6d')]\"]"}, {"name": "timeout", "value": "10"}], "start": 1730061940458, "stop": 1730061940967}, {"name": "Находим элемент", "status": "passed", "parameters": [{"name": "locator", "value": "['xpath', \".//div[contains(@class, 'constructor-element_pos_top')]\"]"}, {"name": "timeout", "value": "10"}], "start": 1730061940967, "stop": 1730061940971}], "parameters": [{"name": "locator_source", "value": "['xpath', \".//a[(@href='/ingredient/61c0c5a71d1f82001bdaaa6d')]\"]"}, {"name": "locator_target", "value": "['xpath', \".//div[contains(@class, 'constructor-element_pos_top')]\"]"}, {"name": "timeout", "value": "10"}], "start": 1730061940458, "stop": 1730061941496}], "start": 1730061940457, "stop": 1730061941496}, {"name": "Кликаем на кнопку оформления заказа", "status": "passed", "steps": [{"name": "Кликаем на элемент", "status": "passed", "steps": [{"name": "Находим элемент", "status": "passed", "parameters": [{"name": "locator", "value": "['xpath', \".//button[text()='Оформить заказ']\"]"}, {"name": "timeout", "value": "10"}], "start": 1730061941496, "stop": 1730061941501}], "parameters": [{"name": "locator", "value": "['xpath', \".//button[text()='Оформить заказ']\"]"}, {"name": "timeout", "value": "10"}], "start": 1730061941496, "stop": 1730061941727}], "start": 1730061941496, "stop": 1730061941727}], "start": 1730061940455, "stop": 1730061941727}, {"name": "Ожидание на отсуствие номера заказа 9999 в окне после создания заказа", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\pages\\main_page.py\", line 57, in wait_invisible_number_9999_in_created_order_window\n    self.invisible_element(Locators.ORDER_NUMBER_9999)\n  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\pages\\base_page.py\", line 91, in invisible_element\n    WebDriverWait(self.driver, timeout).until(expected_conditions.invisibility_of_element_located((locator)))\n  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "steps": [{"name": "Ожидание невидимости элемента", "status": "broken", "statusDetails": {"message": "selenium.common.exceptions.TimeoutException: Message: \n\n", "trace": "  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\.venv\\Lib\\site-packages\\allure_commons\\_allure.py\", line 192, in impl\n    return func(*a, **kw)\n           ^^^^^^^^^^^^^^\n  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\pages\\base_page.py\", line 91, in invisible_element\n    WebDriverWait(self.driver, timeout).until(expected_conditions.invisibility_of_element_located((locator)))\n  File \"C:\\Users\\artmo\\Documents\\3\\pythonProject\\.venv\\Lib\\site-packages\\selenium\\webdriver\\support\\wait.py\", line 105, in until\n    raise TimeoutException(message, screen, stacktrace)\n"}, "parameters": [{"name": "locator", "value": "['xpath', \".//h2[text()='9999']\"]"}, {"name": "timeout", "value": "10"}], "start": 1730061941727, "stop": 1730061952045}], "start": 1730061941727, "stop": 1730061952045}], "parameters": [{"name": "driver", "value": "'firefox'"}], "start": 1730061940455, "stop": 1730061952046, "uuid": "298a5e3d-f34c-4f3d-b09f-e8a807646f0b", "historyId": "ad7939ebd012a68e124893e1e9753f59", "testCaseId": "2fc49f96ba8a150d1f63de3c153b0593", "fullName": "tests.test_account.TestAccount#test_click_history_button_shows_order_history", "labels": [{"name": "parentSuite", "value": "tests"}, {"name": "suite", "value": "test_account"}, {"name": "subSuite", "value": "TestAccount"}, {"name": "host", "value": "DESKTOP-ABCQNV2"}, {"name": "thread", "value": "11168-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test_account"}]}